x-pack/plugins/apm/server/lib/observability_overview/get_transactions_per_minute.ts

```diff
@@ -94,7 +91,7 @@ export async function getTransactionsPerMinute({
     timeseries:
       topTransactionTypeBucket?.timeseries.buckets.map((bucket) => ({
         x: bucket.key,
-        y: bucket.throughput.value,
+        y: calculateThroughput({ start, end, value: bucket.doc_count }),
```

Comment:
We probably want to use the bucketSize here too, but the observability app that provides this option.

Reply:
Yes, using `start`/`end` here will give incorrect results

Reply:
Fixed here: https://github.com/elastic/kibana/pull/112343/commits/fb99e157f7519ca903f59749420f08aa02e06965

Before:
<img width="1372" alt="Screen Shot 2021-09-21 at 17 14 07" src="https://user-images.githubusercontent.com/55978943/134248898-22b3b637-7b89-4577-8efe-e3167c6c80b4.png">

After:
<img width="1366" alt="Screen Shot 2021-09-21 at 17 15 05" src="https://user-images.githubusercontent.com/55978943/134248909-f9a540cb-82dd-4431-b2c5-37b163601aeb.png">